### PR TITLE
Fixes #143, close-bilkent to cose-bilkent; fixes #119, added fcose layout

### LINKED ISSUE #119: "fcose" layout not in the list of possible layouts.
#### Description
Cytospace.js has recently announced about a new type of CoSE family, called fcose. I would like to have fcose in the list of dash-cytospace layouts.

### LINKED ISSUE #143: [BUG] expected an incorrect layout name ("close-bilkent") in 0.3.0
#### Description

I set layout as "cose-bilkent", the layout expect an incorrect layout name (close-bilkent)

#### Steps/Code to Reproduce

Example:

```python
layout= "cose-bilkent"
```

#### Expected Results

according the document, the 'cose-bilkent' is correct name, not close-bilkent

#### Actual Results

```python
Invalid argument `layout.name` passed into Cytoscape with ID "cytoscape_graph".
Expected one of ["random","preset","circle","concentric","grid","breadthfirst","cose","close-bilkent","cola","euler","spread","dagre","klay"].
```

#### Versions

0.3.0

### PR DESCRIPTION
**Pull request by kinimesi**
## About
- Fixing an issue.

## Description of changes 
Fixes #143, replaced close-bilkent with cose-bilkent

## Pre-Merge checklist
- [x ] The project was correctly built with `npm run build:all`.
- [ x] If there was any conflict, it was solved correctly.
- [ ] All changes were documented in CHANGELOG.md.
- [ ] All tests on CircleCI have passed.
- [ ] All Percy visual changes have been approved.
- [ ] Two people have :dancer:'d the pull request. You can be one of these people if you are a Dash Cytoscape core contributor.

## Reference Issues
Closes #143

### CODE REVIEW

**Comment by xhlulu** [comment]
@alexcjohnson Looking at the diff here, really feel like we should do something like the `dev` branch and `main` branch in this repository - just like we are doing with dash-core-components...

File: src/lib/components/Cytoscape.react.js [hunk 1]
```diff
@@ -633,7 +633,7 @@ Cytoscape.propTypes = {
             'grid',
             'breadthfirst',
             'cose',
-            'close-bilkent',
+            'cose-bilkent',
```
Comment (xhlulu) [inline src/lib/components/Cytoscape.react.js:636]: is that the only line you modified before running `npm run build`? just making sure :)

Comment (kinimesi) [reply src/lib/components/Cytoscape.react.js:636]: yes and in a few other places that were mostly documentation. I tested it by setting the layout to `cose-bilkent` and it works.

**Comment by MrTeale** [comment]
Fixes #165

**Pull request merged**